guides/hosting/infrastructure/message-queue.md

```diff
@@ -77,7 +83,7 @@ User=www-data # Change this to webserver's user name
 Restart=always
 # Change the path to your shop path
 WorkingDirectory=/var/www/html
-ExecStart=php /var/www/html/bin/console messenger:consume --time-limit=60 --memory-limit=512M async
+ExecStart=php /var/www/html/bin/console messenger:consume --time-limit=60 --memory-limit=512M async async_low_priority
```

Comment:
do we need to mention that this is only there from 6.5.x? Can you test whant happens when you start a worker for a queue that does not exist?

Maybe we need to split this MR for what already be done in 6.5 and what needs to change in 6.6 (e.g. how the command is executed)

Reply:
I will add a warning that you must not add this in earlier versions as the command will fail :+1: